### PR TITLE
app-shells/bash: Sync with Gentoo upstream; updates to 5.1

### PR DESCRIPTION
**Pull request by sayanchowdhury**
# app-shells/bash: Sync with Gentoo upstream; updates to 5.1

## How to use
```
emerge-amd64-usr app-shells/bash
```

## Testing done

CI passed: http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/3686/cldsv/

To be merged with: https://github.com/flatcar-linux/scripts/pull/155

### CODE REVIEW

File: app-shells/bash/bash-5.0_p18.ebuild [hunk 1]
```diff
@@ -0,0 +1,265 @@
+# Copyright 1999-2021 Gentoo Authors
```
Comment (krnowak) [inline app-shells/bash/bash-5.0_p18.ebuild:1]: Any reason for keeping this ebuild?

File: app-shells/bash/files/bash-5.1-parallel_make.patch [hunk 2]
```diff
@@ -0,0 +1,15 @@
+bashline.c:65:10: fatal error: builtins/builtext.h: No such file or directory
```
Comment (krnowak) [inline app-shells/bash/files/bash-5.1-parallel_make.patch:1]: This patch does not seem to be applied anywhere. Drop it maybe?

File: app-shells/bash/Manifest [hunk 3]
```diff
@@ -1,49 +1,337 @@
...
-DIST bash43-047 4437 SHA256 c69248de7e78ba6b92f118fe1ef47bc86479d5040fe0b1f908ace1c9e3c67c4a SHA512 9332d680ac226491cf8e5986a3261eb76b53ef4b0b2a43405564b088afc0e8658077812c9177bdab7900e05394e69f3047d82ddb59553fc01ef721acfa9d5553 WHIRLPOOL 719fa31d0fb09a75d5abbb63ee5a485802c5eadd06720d6074c2282d117aa6e234f5f75cabd340643710bd14e99260047183386bd1df278c9715aed7b98232d7
-DIST bash43-048 1612 SHA256 5b8215451c5d096ca1e115307ffe6613553551a70369525a0778f216c3a4dfa2 SHA512 7c3963bfbc730b4592668460e86c166e727e9897c1a9235a6860a01bc90d365ef0707f0fac7ed62780dcc84e80dbe8484ec59e9e948def22f44ae03e8a169780 WHIRLPOOL e1a435c3b9fbc0deca80dd89bfd3adcbff1d5a3af50884d2aef54635b8ba1063903e2d99a5a23bd1c64c19c5e9c19de6b479ac777cecdb812555dc5683b8cf71
+DIST bash-2.05b.tar.gz 1956216 BLAKE2B 15962417d13c6eec0db0861b601c33a6b13dc38d41672258f8f9fb8968a000ca1df7d5d3c99bbcb023e0b6b3e703d6c250291ac68ab3f24159eec7c2f0105b43 SHA512 595f84cfd38b239ff37f2c6fc22c669f8e1d4fea34400e30c4079ab657d05f735ee6b0e6af0db6c1b17dbf9cecec81805b467b4dbd45cc97044bfefac731aad6
```
Comment (krnowak) [inline app-shells/bash/Manifest:1]: Could you drop all the lines that are not `bash-5.1.tar.gz` or `bash51-<digits>`?

**Review by krnowak — CHANGES_REQUESTED** [review]

File: app-shells/bash/bash-5.0_p18.ebuild (hunk 1 above)
Comment (sayanchowdhury) [reply app-shells/bash/bash-5.0_p18.ebuild:1]: No reasons, dropping this build.

**Review by dongsupark — APPROVED** [review]
Looks good. Thanks!

File: app-shells/bash/Manifest (hunk 3 above)
Comment (sayanchowdhury) [reply app-shells/bash/Manifest]: I was divided on this. I've dropped as suggested.

File: app-shells/bash/files/bash-5.1-parallel_make.patch (hunk 2 above)
Comment (sayanchowdhury) [reply app-shells/bash/files/bash-5.1-parallel_make.patch]: Sure, I've dropped the other patches as well which were not needed.

**Review by krnowak — APPROVED** [review]
Looks good, thanks.

**Pull request merged**